### PR TITLE
Workaround an issue between React and Google Translate

### PR DESCRIPTION
**Pull request by awaisdar001**
Workaround an issue caused by a known interplay between React and Google Translate that we have no control over. 

[TNL-7867](https://openedx.atlassian.net/browse/TNL-7867)

React issue: https://github.com/facebook/react/issues/11538

### CODE REVIEW

**Review by kdmccormick — COMMENTED** [review]
Just one comment; otherwise, looks good.

@sarina , do you have objections to this change? Although exempting this field from translation isn't super non-English-speaking-friendly, it seems better than the current state of affairs, which is that the entire page breaks if you try to change your country after translating.

### for reference, what happens now:

1. pre-translation:
image

2. post-translation:
image

3. editing the field (note that drop-down has English and Spanish names for each country):
image

4. hit save (salvar), and the page breaks:

image

File: src/account-settings/AccountSettingsPage.jsx [hunk 1]
```diff
@@ -326,38 +326,42 @@ class AccountSettingsPage extends React.Component {
...
-            emptyLabel={
-              this.isEditable('country') ?
-                this.props.intl.formatMessage(messages['account.settings.field.country.empty']) :
-                this.renderEmptyStaticFieldMessage()
-            }
-            isEditable={this.isEditable('country')}
-            {...editableFieldProps}
-          />
-          {showState &&
+          <div className="notranslate">
```
[the remark this answers is not in the document]
Comment (awaisdar001) [reply src/account-settings/AccountSettingsPage.jsx:328]: Sure. Updating the PR

**Comment by sarina** [comment]
No objections, this makes sense.

**Review by kdmccormick — APPROVED** [review]
LGTM. Thanks Awais.

**Comment by awaisdar001** [comment]
Thanks @kdmccormick and @sarina .

**Comment by waheedahmed** [comment]
Hello @awaisdar001, we have experienced this same issue in our Authn MFE app where we are using the Paragon's `StatefulButton`. I guess the accounts app is also using the same `StatefulButton` and causing this problem. I still can see these errors in the NewRelic for accounts app, I have made a fix for this issue in Paragon's `StatefulButton` in this PR https://github.com/edx/paragon/pull/657 and I believe you can remove this workaround with the latest version of Paragon.

Please feel free to reach out if you have any questions.

**Comment by awaisdar001** [comment]
> Hello @awaisdar001, we have experienced this same issue in our Authn MFE app where we are using the Paragon's `StatefulButton`. I guess the accounts app is also using the same `StatefulButton` and causing this problem. I still can see these errors in the NewRelic for accounts app, I have made a fix for this issue in Paragon's `StatefulButton` in this PR [edx/paragon#657](https://github.com/edx/paragon/pull/657) and I believe you can remove this workaround with the latest version of Paragon.
> 
> Please feel free to reach out if you have any questions.

Sounds good @waheedahmed

**Pull request merged**